Watching for file changes with StatReloader
Not Found: /
Not Found: /favicon.ico
Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
MySQLdb.ProgrammingError: (1146, "Table '_livelens.users' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 441, in login
    return LoginView.as_view(**defaults)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/debug.py", line 92, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/generic/edit.py", line 152, in post
    if form.is_valid():
       ^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/forms/forms.py", line 201, in is_valid
    return self.is_bound and not self.errors
                                 ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/forms/forms.py", line 196, in errors
    self.full_clean()
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/forms/forms.py", line 434, in full_clean
    self._clean_form()
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/forms/forms.py", line 455, in _clean_form
    cleaned_data = self.clean()
                   ^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/auth/forms.py", line 250, in clean
    self.user_cache = authenticate(
                      ^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/debug.py", line 42, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/auth/__init__.py", line 77, in authenticate
    user = backend.authenticate(request, **credentials)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/auth/backends.py", line 46, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 54, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 633, in get
    num = len(clone)
          ^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 380, in __len__
    self._fetch_all()
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table '_livelens.users' doesn't exist")
Watching for file changes with StatReloader
Not Found: /api/register
Not Found: /api/auth
Not Found: /register
Not Found: /register/
Not Found: /api/register/
Not Found: /api/register/
Not Found: /register/
Watching for file changes with StatReloader
Not Found: /register/
Bad Request: /api/auth/register/
Internal Server Error: /api/auth/register/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/authapp/views.py", line 37, in post
    self.send_verification_email(user, verification.code)
  File "/Users/levicharles/Documents/liveLens_backend/authapp/views.py", line 88, in send_verification_email
    send_mail(
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/mail/__init__.py", line 87, in send_mail
    return mail.send()
           ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/mail/message.py", line 298, in send
    return self.get_connection(fail_silently).send_messages([self])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/mail/backends/smtp.py", line 127, in send_messages
    new_conn_created = self.open()
                       ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/mail/backends/smtp.py", line 94, in open
    self.connection.login(self.username, self.password)
  File "/Users/levicharles/anaconda3/lib/python3.11/smtplib.py", line 750, in login
    raise last_exception
  File "/Users/levicharles/anaconda3/lib/python3.11/smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "/Users/levicharles/anaconda3/lib/python3.11/smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-450d8006946sm132198555e9.31 - gsmtp')
Bad Request: /api/auth/register/
Unauthorized: /api/auth/login/
Unauthorized: /api/auth/login/
Unauthorized: /api/auth/login/
Unauthorized: /api/auth/login/
Unauthorized: /api/auth/register/
/Users/levicharles/Documents/liveLens_backend/authapp/serializers.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/authapp/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/authapp/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/authapp/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Unauthorized: /api/auth/register/
Internal Server Error: /admin/password_reset/passwordresetcode/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
MySQLdb.ProgrammingError: (1146, "Table '_livelens.password_reset_codes' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1926, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 836, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/views/main.py", line 123, in __init__
    self.get_results(request)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/views/main.py", line 279, in get_results
    result_count = paginator.count
                   ^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/paginator.py", line 93, in count
    return c()
           ^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 608, in count
    return self.query.get_count(using=self.db)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 568, in get_count
    return obj.get_aggregation(using, {"__count": Count("*")})["__count"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 554, in get_aggregation
    result = compiler.execute_sql(SINGLE)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table '_livelens.password_reset_codes' doesn't exist")
Internal Server Error: /admin/authapp/emailverification/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/options.py", line 681, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'status_badge'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 272, in lookup_field
    f = _get_non_gfk_field(opts, name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 303, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/options.py", line 683, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: EmailVerification has no field named 'status_badge'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/test/utils.py", line 112, in instrumented_test_render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/test/utils.py", line 112, in instrumented_test_render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/test/utils.py", line 112, in instrumented_test_render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 336, in result_list
    "results": list(results(cl)),
               ^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 312, in results
    yield ResultList(None, items_for_result(cl, res, None))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 303, in __init__
    super().__init__(*items)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 213, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 281, in lookup_field
    value = attr(obj)
            ^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/authapp/admin.py", line 55, in status_badge
    elif obj.is_expired():
         ^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/authapp/models.py", line 78, in is_expired
    return datetime.now() > self.expires_at
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: can't compare offset-naive and offset-aware datetimes
Unauthorized: /api/auth/verify-email/
Unauthorized: /api/auth/verify-email/
Unauthorized: /api/auth/register/
Unauthorized: /api/auth/login/
Watching for file changes with StatReloader
Unauthorized: /api/auth/register/
Unauthorized: /api/auth/register/
Failed to send verification email to test@example.com: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3a4efe74165sm16194676f8f.53 - gsmtp')
Logged activity: test@example.com - signup
Bad Request: /api/auth/verify-email/
Failed to send verification email to jameshayley190@gmail.com: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-450dc818f27sm118057075e9.18 - gsmtp')
Logged activity: jameshayley190@gmail.com - signup
Unauthorized: /api/auth/register/
Watching for file changes with StatReloader
Verification email sent successfully to levilu18@gmail.com
Logged activity: levilu18@gmail.com - signup
Logged activity: levilu18@gmail.com - profile_update
Unauthorized: /api/auth/login/
Bad Request: /api/auth/login/
/Users/levicharles/Documents/liveLens_backend/authapp/serializers.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/authapp/serializers.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
Logged activity: levilu18@gmail.com - login
Unauthorized: /api/auth/login/
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
Unauthorized: /api/auth/login/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Unauthorized: /api/auth/login/
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Internal Server Error: /api/auth/login
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/deprecation.py", line 122, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/api/auth/login/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
Logged activity: levilu18@gmail.com - login
Internal Server Error: /api/password-reset/request/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 76, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
MySQLdb.ProgrammingError: (1146, "Table '_livelens.password_reset_passwordresetrequest' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/password_reset/views.py", line 31, in post
    reset_request = serializer.save(
                    ^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/password_reset/serializers.py", line 30, in save
    ).update(is_used=True)
      ^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1259, in update
    rows = query.get_compiler(self.db).execute_sql(ROW_COUNT)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 2060, in execute_sql
    row_count = super().execute_sql(result_type)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 76, in execute
    return self.cursor.execute(query, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/MySQLdb/connections.py", line 255, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table '_livelens.password_reset_passwordresetrequest' doesn't exist")
Password reset email sent successfully to levilu18@gmail.com
Logged activity: levilu18@gmail.com - password_change
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Unauthorized: /api/auth/register/
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
Watching for file changes with StatReloader
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/auth/register/
Bad Request: /api/auth/register/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/auth/register/
Verification email sent successfully to levilu18@gmail.com
Logged activity: levilu18@gmail.com - signup
Unauthorized: /api/auth/verify-email/
Unauthorized: /api/auth/verify-email/
Logged activity: levilu18@gmail.com - profile_update
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Password reset email sent successfully to levilu18@gmail.com
Logged activity: levilu18@gmail.com - password_change
Logged activity: levilu18@gmail.com - login
Unauthorized: /api/auth/profile/
Not Found: /api/auth/profile/3
Not Found: /api/auth/profile/3/
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/urls.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /api/auth/profile/3/
Unauthorized: /api/auth/profile/
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/password_reset/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/password_reset/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/password_reset/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/password_reset/models.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /api/auth/profile/
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/app/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Internal Server Error: /admin/media_content/content/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 246, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 867, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/views/main.py", line 91, in __init__
    self.root_queryset = model_admin.get_queryset(request)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/media_content/admin.py", line 239, in get_queryset
    return super().get_queryset(request).select_related('author').annotate(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1645, in annotate
    return self._annotate(args, kwargs, select=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1697, in _annotate
    clone.query.add_annotation(
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1218, in add_annotation
    annotation = annotation.resolve_expression(self, allow_joins=True, reuse=None)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 181, in resolve_expression
    result = super().resolve_expression(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 63, in resolve_expression
    c = super().resolve_expression(query, allow_joins, reuse, summarize)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 298, in resolve_expression
    source_expressions = [
                         ^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 300, in <listcomp>
    expr.resolve_expression(query, allow_joins, reuse, summarize)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 902, in resolve_expression
    return query.resolve_ref(self.name, allow_joins, reuse, summarize)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 2046, in resolve_ref
    join_info = self.setup_joins(
                ^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1897, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
                                       ^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1802, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'mediainteraction' into field. Choices are: author, author_id, average_rating, banner, category, collections, comment_count, content_type, created_at, creator, description, difficulty_level, download_count, duration, episode_number, file_size, id, is_featured, is_live, is_premium, is_trending, language, like_count, live_stream_url, poster, published_at, rating_count, release_year, scheduled_live_time, series_name, short_description, slug, status, subtitles_available, tags, thumbnail, title, trailer_duration, trailer_file, updated_at, video_file, video_quality, view_count
Internal Server Error: /admin/media_content/film/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 246, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 867, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/views/main.py", line 91, in __init__
    self.root_queryset = model_admin.get_queryset(request)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/media_content/admin.py", line 85, in get_queryset
    return super().get_queryset(request).select_related('author').annotate(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1645, in annotate
    return self._annotate(args, kwargs, select=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1697, in _annotate
    clone.query.add_annotation(
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1218, in add_annotation
    annotation = annotation.resolve_expression(self, allow_joins=True, reuse=None)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 181, in resolve_expression
    result = super().resolve_expression(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 63, in resolve_expression
    c = super().resolve_expression(query, allow_joins, reuse, summarize)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 298, in resolve_expression
    source_expressions = [
                         ^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 300, in <listcomp>
    expr.resolve_expression(query, allow_joins, reuse, summarize)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 902, in resolve_expression
    return query.resolve_ref(self.name, allow_joins, reuse, summarize)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 2046, in resolve_ref
    join_info = self.setup_joins(
                ^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1897, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
                                       ^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1802, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'mediainteraction' into field. Choices are: author, author_id, average_rating, banner, box_office, budget, cast, category, collections, comment_count, created_at, description, director, download_count, duration, episode_number, file_size, id, is_featured, is_premium, is_series, is_trending, language, like_count, mpaa_rating, poster, producer, published_at, rating_count, release_year, season_number, series_name, short_description, slug, status, studio, subtitles_available, tags, thumbnail, title, trailer_duration, trailer_file, updated_at, video_file, video_quality, view_count
Internal Server Error: /admin/media_content/content/
Traceback (most recent call last):
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 246, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 867, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/contrib/admin/views/main.py", line 91, in __init__
    self.root_queryset = model_admin.get_queryset(request)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/media_content/admin.py", line 239, in get_queryset
    return super().get_queryset(request).select_related('author').annotate(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1645, in annotate
    return self._annotate(args, kwargs, select=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1697, in _annotate
    clone.query.add_annotation(
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1218, in add_annotation
    annotation = annotation.resolve_expression(self, allow_joins=True, reuse=None)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 181, in resolve_expression
    result = super().resolve_expression(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/aggregates.py", line 63, in resolve_expression
    c = super().resolve_expression(query, allow_joins, reuse, summarize)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 298, in resolve_expression
    source_expressions = [
                         ^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 300, in <listcomp>
    expr.resolve_expression(query, allow_joins, reuse, summarize)
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/expressions.py", line 902, in resolve_expression
    return query.resolve_ref(self.name, allow_joins, reuse, summarize)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 2046, in resolve_ref
    join_info = self.setup_joins(
                ^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1897, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
                                       ^^^^^^^^^^^^^^^^^^^
  File "/Users/levicharles/Documents/liveLens_backend/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1802, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'mediainteraction' into field. Choices are: author, author_id, average_rating, banner, category, collections, comment_count, content_type, created_at, creator, description, difficulty_level, download_count, duration, episode_number, file_size, id, is_featured, is_live, is_premium, is_trending, language, like_count, live_stream_url, poster, published_at, rating_count, release_year, scheduled_live_time, series_name, short_description, slug, status, subtitles_available, tags, thumbnail, title, trailer_duration, trailer_file, updated_at, video_file, video_quality, view_count
/Users/levicharles/Documents/liveLens_backend/media_content/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/media_content/admin.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
Logged activity: levilu18@gmail.com - login
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
Watching for file changes with StatReloader
/Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
INFO 2025-06-07 12:00:04,844 autoreload 89430 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:00:20,336 autoreload 89430 140704394378688 /Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
INFO 2025-06-07 12:00:20,913 autoreload 89463 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:00:26,290 autoreload 89463 140704394378688 /Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
INFO 2025-06-07 12:00:26,820 autoreload 89488 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:01:28,797 autoreload 89488 140704394378688 /Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
INFO 2025-06-07 12:01:29,433 autoreload 89641 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:20:35,108 basehttp 89641 123145539342336 "GET /admin/authapp/user/ HTTP/1.1" 200 30798
WARNING 2025-06-07 12:20:35,810 log 89641 123145539342336 Not Found: /favicon.ico
WARNING 2025-06-07 12:20:35,811 basehttp 89641 123145539342336 "GET /favicon.ico HTTP/1.1" 404 3829
INFO 2025-06-07 12:20:36,607 basehttp 89641 123145539342336 "GET /admin/authapp/user/ HTTP/1.1" 200 30798
INFO 2025-06-07 12:20:36,639 basehttp 89641 123145539342336 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,645 basehttp 89641 123145589710848 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,650 basehttp 89641 123145539342336 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,655 basehttp 89641 123145572921344 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,656 basehttp 89641 123145623289856 "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,658 basehttp 89641 123145606500352 "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,659 basehttp 89641 123145589710848 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,659 basehttp 89641 123145539342336 "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,661 basehttp 89641 123145572921344 "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,667 basehttp 89641 123145539342336 "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,668 basehttp 89641 123145623289856 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,668 basehttp 89641 123145589710848 "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,668 basehttp 89641 123145606500352 "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,668 basehttp 89641 123145572921344 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,675 basehttp 89641 123145556131840 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:36,689 basehttp 89641 123145623289856 "GET /static/admin/img/search.svg HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,689 basehttp 89641 123145606500352 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,714 basehttp 89641 123145606500352 "GET /static/admin/img/icon-no.svg HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,714 basehttp 89641 123145623289856 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:36,714 basehttp 89641 123145556131840 "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2025-06-07 12:20:38,737 basehttp 89641 123145623289856 "GET /admin/authapp/user/ HTTP/1.1" 200 30798
INFO 2025-06-07 12:20:38,781 basehttp 89641 123145623289856 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:42,976 basehttp 89641 123145623289856 "GET /admin/animations/aianimationrequest/ HTTP/1.1" 200 26057
INFO 2025-06-07 12:20:43,017 basehttp 89641 123145623289856 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:45,617 basehttp 89641 123145623289856 "GET /admin/animations/animationcollection/ HTTP/1.1" 200 24724
INFO 2025-06-07 12:20:45,666 basehttp 89641 123145623289856 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:47,462 basehttp 89641 123145623289856 "GET /admin/animations/animationinteraction/ HTTP/1.1" 200 24592
INFO 2025-06-07 12:20:47,495 basehttp 89641 123145623289856 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:49,201 basehttp 89641 123145539342336 "GET /admin/animations/animationplaylist/ HTTP/1.1" 200 25028
INFO 2025-06-07 12:20:49,245 basehttp 89641 123145539342336 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:52,914 basehttp 89641 123145539342336 "GET /admin/animations/animationview/ HTTP/1.1" 200 24436
INFO 2025-06-07 12:20:52,952 basehttp 89641 123145539342336 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:20:54,522 basehttp 89641 123145539342336 "GET /admin/animations/animation/ HTTP/1.1" 200 29685
INFO 2025-06-07 12:20:54,564 basehttp 89641 123145539342336 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-06-07 12:35:33,986 autoreload 89641 140704394378688 /Users/levicharles/Documents/liveLens_backend/app/settings.py changed, reloading.
INFO 2025-06-07 12:35:35,027 autoreload 91801 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:40:03,462 autoreload 91801 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:40:04,191 autoreload 92305 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:43:00,552 autoreload 92305 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:43:01,478 autoreload 92552 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:43:53,698 autoreload 92552 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:43:54,691 autoreload 92672 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:43:59,566 autoreload 92672 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:00,071 autoreload 92698 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:02,392 autoreload 92698 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:03,114 autoreload 92718 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:04,629 autoreload 92718 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:05,328 autoreload 92729 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:08,894 autoreload 92729 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:09,656 autoreload 92752 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:14,268 autoreload 92752 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:14,844 autoreload 92776 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:19,197 autoreload 92776 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:19,856 autoreload 92794 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:43,434 autoreload 92794 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:44,413 autoreload 92853 140704394378688 Watching for file changes with StatReloader
INFO 2025-06-07 12:44:45,832 autoreload 92853 140704394378688 /Users/levicharles/Documents/liveLens_backend/comments/apps.py changed, reloading.
INFO 2025-06-07 12:44:46,404 autoreload 92864 140704394378688 Watching for file changes with StatReloader
